cmd/oras/internal/display/progress/status.go

```diff
@@ -147,15 +148,16 @@ func (s *status) String(width int) (string, string) {
 
 // calculateSpeed calculates the speed of the progress and update last status.
 // caller must hold the lock.
-func (s *status) calculateSpeed() bytes {
+func (s *status) calculateSpeed() humanize.Bytes {
 	now := time.Now()
 	secondsTaken := now.Sub(s.lastRenderTime).Seconds()
+	secondsTaken = math.Max(secondsTaken, float64(bufFlushDuration.Milliseconds())/1000)
```

Comment:
Instead of using `math.Max`, can you use `if secondsTaken == 0` instead as it is an extremely rare case?